DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -55,5 +58,19 @@ public UserService(IUserDataService userDataService)
 
             return (adminUser, delegateUser);
         }
+
+        public List<CentreUserDetails> GetAvailableCentres(AdminUser? adminUser, List<DelegateUser> delegateUsers)
+        {
+            var availableCentres = delegateUsers.Select(du => new CentreUserDetails
```

Comment:
Super minor, but could we reformat this such that .Select is on a new line and the .List is on a new line to make it easier to read?